### DISCUSSION TITLE
Turning a private squad public

### DISCUSSION THREAD

**Discussion by avi12** [dailydotdev/daily#1548] [General]
I'm in a [private squad](https://dly.to/nXmahU9GnCt) that I wish to turn public. I'm one of the admins and over a month ago I started the process of making it public, but it got stuck somewhere down the line

**Comment by nimrodkra** [comment]
Hi Avi, making your squad public does no longer require to go through
review. You can head to your Squad settings and change it to public. Good
luck!
